pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/VarByteChunkForwardIndexReaderV4.java

```diff
@@ -81,6 +80,12 @@ public VarByteChunkForwardIndexReaderV4(PinotDataBuffer dataBuffer, FieldSpec.Da
     _isSingleValue = isSingleValue;
   }
 
+  public void validateIndexVersion(PinotDataBuffer dataBuffer) {
```

Comment:
I meant we can add a `getVersion()` method into this class, and override it in v5 reader